docs: separate README file for each package (#74)
pubnub/react-chat-components: packages/react/README.md

# PubNub Chat Components for React

> PubNub Chat Components for React are the fastest way to add chat features like direct and group messaging, typing indicators, reactions, without going through the complexity of low-level architecture of realtime networks.

Reviewer: ```suggestion
PubNub Chat Components for React are the fastest way to add chat features like direct and group messaging, typing indicators, or reactions, without going through the complexity of the low-level architecture of real-time networks.
```

Author: fixed

> - **Reduced Implementation Time**: Develop proof-of-concept and production ready apps faster using predefined components.

Reviewer: ```suggestion
- **Reduced Implementation Time**: Develop proof-of-concept and production-ready apps faster using predefined components.
```

> - **Fast and Simple Extensibility**: Add rich features like typing indicators, read receipts, reactions etc. without writing complex code.

Reviewer: ```suggestion
- **Fast and Simple Extensibility**: Add rich features, like typing indicators, read receipts, or reactions, without writing complex code.
```

Author: fixed

- **Flexible and Customizable Components**: Customize component design and add custom components to extend functionality.
- **High Scalability**: Let PubNub take care of scaling and reliability as you grow your app.
- **Easy Theming**: Use the built-in light and dark themes or create custom ones for various use cases: group, support, and event chats.
- **Strong Typing**: Utilize the power of TypeScript to develop your application.

> > You can also check out our [PubNub Chat Components for React Native](https://www.npmjs.com/package/@pubnub/react-native-chat-components)

Reviewer: ```suggestion
```

> ![PubNub Chat Components](screenshot-desktop.png)

Reviewer: ```suggestion
![PubNub Chat Components](screenshot-desktop.png)

> Apart from PubNub Chat Components for React, we provide a separate npm package for React Native. Check out our [PubNub Chat Components for React Native](https://www.npmjs.com/package/@pubnub/react-native-chat-components) for more details.
```

## Features

- **User and Channel Metadata**: Add additional information about the users, channels, and their memberships from PubNub Objects storage using custom hooks
- **Subscriptions**: Subscribe to user channels automatically
- **Messages**: Publish and display new and historical text messages

> - **Presence**: Get currently active users, observe their state, and notify about changes

Reviewer: ```suggestion
- **Presence**: Get currently active users, observe their state, and notify them about changes
```

> - **Presence**: Get currently active users, observe their state, and notify about changes

Reviewer: ```suggestion
- **Presence**: Get currently active users, observe their state, and notify them about changes
```
